pkg/api/node/formatter.go

```diff
@@ -145,11 +151,40 @@ func (h ActionHandler) do(rw http.ResponseWriter, req *http.Request) error {
 			return apierror.NewAPIError(validation.InvalidBodyContent, fmt.Sprintf("Failed to decode request body: %v ", err))
 		}
 		return h.powerAction(toUpdate, input.Operation)
+	case enableCPUManager:
+		return h.enableCPUManager(toUpdate)
```

Comment:
Probably, we can use `apierror.NewAPIError(validation.InvalidAction, err.Error())` as returned error.

Reply:
Thanks for the comment @FrankYang0529, I modify the error handling a bit in https://github.com/harvester/harvester/pull/6190/files#diff-eb153a88748df1fbf46481b843d26f5882036ceca1c94824bf9fecfcd649c862R107
Now the error response from apiserver will be take care, so the error status code in validator should be correctly handled and return to the client. Do you think this approach is valid ?

Reply:
Yes, sounds good to me.